### PR TITLE
[ci] Push images to diem/ ECRs

### PR DESCRIPTION
**Pull request by mgorven**
Test Plan:

    $ dockerhub_to_novi_ecr.sh -t master_76576974 -r <ECR URL>

### CODE REVIEW

**Review by rexhoffman — APPROVED** [review]
Should work, can be tested by creating a **test-#** branch in libra/diem if you want.

File: docker/dockerhub_to_novi_ecr.sh [hunk 1]
```diff
@@ -60,25 +60,8 @@ docker pull --disable-content-trust=false docker.io/libra/client:"$DOCKERHUB_TAG
...
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_safety_rules:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_cluster_test:"$OUTPUT_TAG"
-docker push --disable-content-trust=true "$TARGET_REPO"/libra_client:"$OUTPUT_TAG"
-
-
-#Push the proper locations to novi ecr.
+# Push to old Novi ECR
 docker tag libra/init:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra/init:"$OUTPUT_TAG"
 docker tag libra/faucet:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra/faucet:"$OUTPUT_TAG"
 docker tag libra/tools:"$DOCKERHUB_TAG" "$TARGET_REPO"/libra/tools:"$OUTPUT_TAG"
```
Comment (sausagee) [inline docker/dockerhub_to_novi_ecr.sh:67]: yikes, I just landed https://github.com/libra/diem/pull/6846 this afternoon which removed the libra/* images. I guessed we still need them downstream.

**Review by sausagee — APPROVED** [review]

File: docker/dockerhub_to_novi_ecr.sh (hunk 1 above)
Comment (mgorven) [reply docker/dockerhub_to_novi_ecr.sh:67]: I haven't switched the ECR repos yet, so yes

**Comment by rexhoffman** [comment]
@mgorven this can be closed now, currently pushing to dockerhub diem, docker libra, and ecr.

**Pull request closed without merging**